src/main/java/ru/jma/userservice/controller/InteractionController.java

```diff
@@ -0,0 +1,38 @@
+package ru.jma.userservice.controller;
+
+import com.netflix.discovery.EurekaClient;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.core.publisher.Mono;
+import ru.jma.dto.swagger.MessageDto;
+
+import java.time.LocalDateTime;
+
+@RestController
+public class InteractionController {
```

Comment:
Use lombok for constructors
inject courses uri as @Value
It may require additional lombok config via .config file (google lombok copyable annotations)